src/axom/primal/tests/primal_clip.cpp

```diff
@@ -2129,6 +2129,48 @@ TEST(primal_clip, polygon_intersect_robustness)
   }
 }
 
+TEST(primal_clip, polygon_clip_failure)
```

Comment:
Presumably, this was a case that used to fail.
Would `polygon_clip_regression` be a better name than `_failure` ?

Reply:
Renamed it.